integration-web/src/main/resources/log4j2.xml

```diff
@@ -7,6 +7,9 @@
         <Property name="log4j2.symphonyLogLevel">INFO</Property>
         <Property name="log4j2.consoleLogThreshold">INFO</Property>
         <Property name="log4j2.outputAllToConsole">false</Property>
+        <Property name="symphony.instanceId">integration-bridge</Property>
+        <Property name="symphony.maxLogsPerBulkRequest">10</Property>
+        <Property name="symphony.maxPayloadSize">100000</Property>
```

Comment:
We had to put this configuration in place due to issue https://perzoinc.atlassian.net/browse/CP-1023. We have a ticket to track this and adjust if necessary in the future.